ts/navigation/ProfileNavigator.tsx

```diff
@@ -42,6 +43,11 @@ const ProfileStackNavigator = () => (
     initialRouteName={ROUTES.PROFILE_DATA}
     screenOptions={{ gestureEnabled: isGestureEnabled, headerMode: "screen" }}
   >
+    <Stack.Screen
+      options={{ headerShown: true }}
+      name={ROUTES.SETTINGS_MAIN}
+      component={ProfileMainScreen}
+    />
```

Comment:
Maybe this too it should depend on the FF? What do you think?

Reply:
We only navigate this route when the FF is active so there is no risk of navigating it in other situations, but it makes sense to me that we can add it

Reply:
done!